### ISSUE TITLE
Can't edit Width, Height, Margin

### ISSUE THREAD

**Issue by dubeg** [MarkHopper24/barcodrod.io#17]
In Create, I can't edit the Width, Height nor the margin. Is it a [focus issue](https://stackoverflow.com/questions/39096758/cant-enter-enter-text-in-textbox-control-inside-flyout)?

![image](https://github.com/MarkHopper24/barcodrod.io/assets/6020602/a380512d-4830-465d-be53-5cc84b1b0f75)

**Comment by dubeg** [comment]
Since the issue is related to the flyout, I can just fullscreen the application to work around this.

But I tested setting **AllowFocusOnInteraction="True"** on `AppBarElementContainer`, and it seems to fix the problem.

```
<AppBarElementContainer HorizontalAlignment="Center" AllowFocusOnInteraction="True">
    <StackPanel Orientation="Horizontal" Margin="0,5,10,0" >
        <TextBlock Style="{ThemeResource BaseTextBlockStyle}" FontSize="12" Text="Width:" Margin="0,7,5,0" />
        <TextBox x:Name ="userWidth" PlaceholderText="800"  />
    </StackPanel>
</AppBarElementContainer>
<AppBarElementContainer HorizontalAlignment="Center" AllowFocusOnInteraction="True">
    <StackPanel Orientation="Horizontal" Margin="0,5,10,0">
        <TextBlock Style="{ThemeResource BaseTextBlockStyle}" FontSize="12" Text="Height:" Margin="0,7,5,0" />
        <TextBox x:Name="userHeight"  PlaceholderText="800" HorizontalAlignment="Stretch"/>
    </StackPanel>
</AppBarElementContainer>
```

**Comment by MarkHopper24** [comment]
Thanks for reporting this @dubeg. Worked on this issue earlier this week and found the same cause & fix. Will be addressed in the next update.

**Comment by MarkHopper24** [comment]
This issue has been resolved in v1.7 which has been submitted to the Microsoft Store for review. I expect it to be available in the next day or so.

![image](https://github.com/MarkHopper24/barcodrod.io/assets/106134632/e33e4644-da25-4aa0-a1cc-d7600c1d1c21)

@dubeg Thanks again for the report. Closing this out for now.
